scss/widgets/material/slider/_index.scss

```diff
@@ -88,6 +88,7 @@ $material-slider-disabled-tooltip-top-margin: 5px;
       }
 
       .dx-popup-content {
+        overflow: hidden;
```

Comment:
How can we test this?

Reply:
This change for 23.1 version only to prevent scrollbar showing. In 23.2 we redesigned the tooltip and this issue not relevant here and we already test tooltip appearance there. Also we plan to run screenshot testing of our demos in Material & Fluent themes.